test/onnx/onnx_test_common.py

```diff
@@ -140,3 +154,199 @@ def _run_test(m, remained_onnx_input_idx, flatten=True, ignore_none=True):
             )
         if not is_model_script and not self.is_script:
             _run_test(model, tracing_remained_onnx_input_idx)
+
+    @_beartype.beartype
+    def run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
```

Comment:
Does this need to be a method in the test class or can it be a global function? Prefer a global function if it does not require states in self?

Reply:
hmmm thought about that, but the function requires test_suite attributes: op_level_debug and dynamic_shapes. We could move it globally, but will need to impot class TestFx... to annotate, which is cons.

Reply:
SG! This is good then imo